janusgraph-codepipelines-ci/README.md

```diff
@@ -0,0 +1,57 @@
+#JanusGraph CodePipelines CI
+CodePipelines CI is a mechanism JanusGraph can use to do release testing in massively
+parallel fashion (to the extent of AWS CodePipelines and CodeBuild service limits).
+
+## Steps to kick off a build
+Follow the procedure below to run all release testing in parallel.
+This procedure requires you to have an AWS account.
+It also requires you to create two service roles in IAM: one for CodePipeline and
+one for CodeBuild.
+
+1. Get a personal access token from [GitHub](https://github.com/settings/tokens) with repo and admin:repo_hook scopes.
```

Comment:
I've confirmed in a [use case](https://github.com/stelligent/dromedary/blame/master/README.md#L77) and The [documentation](http://docs.aws.amazon.com/codepipeline/latest/userguide/integrations-action-type.html) only mentions the repo scope but it assumes the user will create the pipeline using the console. To create the pipeline programmatically, the admin:repo_hook scope is required.